packages/input/src/index.vue

```diff
@@ -0,0 +1,471 @@
+<template>
+  <div
+    :class="[
+      type === 'textarea' ? 'el-textarea' : 'el-input',
+      inputSize ? 'el-input--' + inputSize : '',
+      {
+        'is-disabled': inputDisabled,
+        'is-exceed': inputExceed,
+        'el-input-group': $slots.prepend || $slots.append,
+        'el-input-group--append': $slots.append,
+        'el-input-group--prepend': $slots.prepend,
+        'el-input--prefix': $slots.prefix || prefixIcon,
+        'el-input--suffix': $slots.suffix || suffixIcon || clearable || showPassword
+      }
+    ]"
+    @mouseenter="hovering = true"
+    @mouseleave="hovering = false"
+  >
+    <template v-if="type !== 'textarea'">
+      <!-- 前置元素 -->
+      <div v-if="$slots.prepend" class="el-input-group__prepend">
+        <slot name="prepend"></slot>
+      </div>
+      <input
+        v-if="type !== 'textarea'"
+        ref="input"
+        class="el-input__inner"
+        v-bind="$attrs"
```

Comment:
Hello, the `$attrs` maybe make a error, its behavior in below.

Incorrect behavior:
![image](https://user-images.githubusercontent.com/33176053/91642688-f287cc00-ea5f-11ea-8e67-ad9ae852388e.png)

Correct behavior:
![image](https://user-images.githubusercontent.com/33176053/91642727-3d094880-ea60-11ea-8cc2-6251012148e1.png)


The `$attrs` pass the `class` attr of `el-input` component to `input`. `style` attr will be same behavior.

If delete this line directly, the `placeholder` will also not be passed.

Delete this line:
![image](https://user-images.githubusercontent.com/33176053/91642758-89548880-ea60-11ea-898b-708649340c9c.png)